Header header
int16 tag_id
geometry_msgs/Point position
geometry_msgs/Quaternion orientation